client/landing/stepper/declarative-flow/internals/steps-repository/intro/intro.tsx

```diff
@@ -42,7 +42,7 @@ const Intro: React.FC< Props > = ( { goNext, flowName } ) => {
 			<h1 className="intro__title">
 				<span>{ introContent[ flowName ].title }</span>
 			</h1>
-			<Button className="intro__button" primary onClick={ goNext }>
+			<Button className="intro__button" primary onClick={ onSubmit }>
```

Comment:
Is there a specific reason to 'navigate' over the flows that are not gathering data via submit instead of 'goNext'?
Not questioning the implementation, it is just for my better understanding =)